dubbo-compatible/src/main/java/com/alibaba/dubbo/rpc/Invoker.java

```diff
@@ -95,5 +95,15 @@ public void destroy() {
         public org.apache.dubbo.rpc.Invoker<T> getOriginal() {
             return invoker;
         }
+        
+        @Override
+        public int hashCode() {
+            return invoker.hashCode();
+        }
+        
+        @Override
+        public boolean equals(Object o) {
+            return invoker.equals(o);
```

Comment:
Check if target object's type is `alibaba invoker` would be better?

Reply:
> Check if target object's type is `alibaba invoker` would be better?

done